lib/mpl_toolkits/tests/test_axes_grid1.py

```diff
@@ -406,3 +406,18 @@ def test_image_grid():
     for i in range(4):
         grid[i].imshow(im)
         grid[i].set_title('test {0}{0}'.format(i))
+
+
+def test_gettightbbox():
```

Comment:
Yeah, sorry, its not a very good test.  But I'm not sure why axes_grid does that, so that's a different issue.  The issue here was that the png being made was not being cropped properly to include the big box when `bbox_inches='tight'` was used.